instrumentation/base/lib/opentelemetry/instrumentation.rb

```diff
@@ -7,6 +7,7 @@
 require 'opentelemetry'
 require 'opentelemetry-registry'
 require 'opentelemetry/instrumentation/base'
+require 'opentelemetry/instrumentation/metrics_patch' if defined?(OpenTelemetry::Metrics) # maybe also add Env var check?
```

Comment:
I'm thinking an environment variable check might be handy at this stage because the config gets registered later, so I wouldn't want to check for a `:send_metrics` config value here, whereas environment variables should be easily accessible.